build/karma.conf.js

```diff
@@ -0,0 +1,23 @@
+// we can just use the exact same webpack config by requiring it
+// but make sure to delete the normal entry
+var webpackConf = require('./webpack.base.conf')
+delete webpackConf.entry
+
+module.exports = function (config) {
```

Comment:
- added tests for element tag name
- for aria checked attribute there is a test https://github.com/vuikit/vuikit/pull/18/files#diff-b143da5a85bf7b3ed6f655995fd5e983R37, do you mean something else?